vite.config.ts

```diff
@@ -1,15 +1,31 @@
 import { defineConfig } from 'vite';
 import minifyHTML from 'rollup-plugin-minify-html-literals';
+import { Config, generateBlockEntries } from './vite.helpers';
 
 const { resolve } = require('path');
 
 const isProd = process.env.NODE_ENV === 'production';
 
-// @ts-ignore:next-line
+const config: Config = {
+  mainTsPath: 'src/main.ts',
+  mainScssPath: 'src/styles/sass/main.scss',
+  fontsScssPath: 'src/styles/sass/fonts.scss',
+  lazyStylesScssPath: 'src/styles/sass/lazy-styles.scss',
+  blocksName: ['counter', 'banner', 'features', 'posts'],
```

Comment:
ich hab noch ne idee - man könnte das Array auch dynamisch erstellen, einfach über ein node-script das alle Ordner Namen aus dem "Blocks" ordner ausließt und in ein Array schreibt. Somit hätte man keinen manuellen Pflegeaufwand mehr.

Reply:
ich finde das gut. Ich versuche es zu implementieren.

Reply:
resolved with https://github.com/ifahrentholz/eds-editorial/pull/29/commits/fbb124db748d85fb0d4ab9c877d477218e32a12e